akka-cluster/src/test/scala/akka/cluster/GossipSpec.scala

```diff
@@ -256,6 +256,22 @@ class GossipSpec extends AnyWordSpec with Matchers {
       state(g, dc2c1).convergence(Set.empty) should ===(false)
     }
 
+    "not reach convergence for first member of other data center until all have seen the gossip" in {
+      val dc2e1 = TestMember(e1.address, status = Joining, roles = Set.empty, dataCenter = "dc2")
+      val g = Gossip(members = SortedSet(dc1a1, dc1b1, dc2e1)).seen(dc1a1.uniqueAddress).seen(dc2e1.uniqueAddress)
+      // dc1b1 has not seen the gossip
+
+      // dc1 hasn't reached convergence
+      state(g, dc1a1).convergence(Set.empty) should ===(false)
```

Comment:
why can't dc1 have convergence? leader actions could safely happen in that dc right?

Reply:
because dc1b1 hasn't marked it as seen

I added that to the comment